app/presenters/statistics_announcement_presenter.rb

```diff
@@ -0,0 +1,58 @@
+class StatisticsAnnouncementPresenter < ContentItemPresenter
+  include ActionView::Helpers::UrlHelper
+
+  def from
+    content_item["links"]["organisations"].map { |org|
+      link_to(org["title"], org["base_path"])
+    }
+  end
+
+  def part_of
+    content_item["links"]["policy_areas"].map { |policy_area|
+      link_to(policy_area["title"], policy_area["base_path"])
+    }
+  end
+
+  def release_date
+    content_item["details"]["display_date"]
+  end
+
+  def release_date_and_status
+    return "#{release_date} (#{state})" unless cancelled?
+    release_date
+  end
+
+  def other_metadata
+    if cancelled?
+    {
```

Comment:
Can we indent this hash one level, I thought I was looking at JavaScript for a second :eyeglasses: